### DISCUSSION TITLE
How to hide share button in the embedded video?

### DISCUSSION THREAD

**Discussion by abdullahqureshi1994** [mediacms-io/mediacms#746] [Q&A]
Hi there,

Can anyone let me know how to hide the share button in the embedded video?

**Comment by jp-rad** [comment]
There are no setting parameters to disable, so edit `VideoViewer` source code and rebuild `frontend` .
https://github.com/mediacms-io/mediacms/tree/main/frontend

1. `cd /home/mediacms.io/mediacms/frontend`
1. `npm install`
1. Comment out the line `bottomRight: ...` :
https://github.com/mediacms-io/mediacms/blob/main/frontend/src/static/js/components/media-viewer/VideoViewer/index.js#L281
1. `npm run dist`
1. `cp -rf /home/mediacms.io/mediacms/frontend/dist/static /home/mediacms.io/mediacms/`
1. Reload the embedded page. Before doing so, clear your browser's cache file.

**Comment by abdullahqureshi1994** [comment]
@jp-rad Thanks a lot.
I deployed it using docker can you guide me on the commands regarding that?

**Comment by jp-rad** [comment]
Suppose you are installing in the following way:
https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md#3-docker-installation

Edit /_path/to/mediacms/installation_/frontend/src/static/js/components/media-viewer/VideoViewer/index.js

>
> Comment out the line bottomRight: ... :
> https://github.com/mediacms-io/mediacms/blob/main/frontend/src/static/js/components/media-viewer/VideoViewer/index.js#L281
>

Connect docker service:
$ `docker exec -it mediacms_web_1 /bin/bash`

Rebuild:

```

apt update
apt upgrade -y
apt install npm -y

cd /home/mediacms.io/mediacms/frontend
npm install
npm run dist
cp -rf /home/mediacms.io/mediacms/frontend/dist/static /home/mediacms.io/mediacms/

```

Reload the embedded page. Before doing so, clear your browser's cache file.

**Comment by abdullahqureshi1994** [comment]
Thanks @jp-rad
